synapse/handlers/auth.py

```diff
@@ -569,6 +574,12 @@ def check_user_exists(self, user_id):
             defer.Deferred: (unicode) canonical_user_id, or None if zero or
```

Comment:
Please could you add a note to the docstrings of the methods in here which do ratelimiting that they will do ratelimiting?